### PR TITLE
Create models and migrate

### PR DESCRIPTION
**Pull request by Nazarcheck**

### CODE REVIEW

File: lesson5/app/models/patient.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Patient < ApplicationRecord
+    has_many :appointments
+    has_many :physicians, through: :appointments
+  end
```
Comment (olehvavryniv) [inline lesson5/app/models/patient.rb]: Add EOF

File: lesson5/app/models/appointment.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+class Appointment < ApplicationRecord
+    belongs_to :physician
+    belongs_to :patient
+  end
```
Comment (olehvavryniv) [inline lesson5/app/models/appointment.rb]: Add EOF

**Review by olehvavryniv — CHANGES_REQUESTED** [review]

File: lesson5/app/models/patient.rb (hunk 1 above)
Comment (olehvavryniv) [inline lesson5/app/models/patient.rb]: Please fix indentation here.

**Comment by Alice932** [comment]
All is done, IMHO

File: lesson5/app/models/physician.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class Physician < ApplicationRecord
+    has_many :appointments
+    has_many :patients, through: :appointments
+end
+
```
Comment (olehvavryniv) [inline lesson5/app/models/physician.rb]: Newline still not fixed here.

File: lesson5/db/migrate/20211214151820_create_appointments.rb [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    end
+
+    create_table :appointments do |t|
+      t.belongs_to :physician
+      t.belongs_to :patient
+      t.datetime :appointment_date
+      t.timestamps
+    end
+  end
+end
```
Comment (olehvavryniv) [inline lesson5/db/migrate/20211214151820_create_appointments.rb]: And here.

**Review by olehvavryniv — CHANGES_REQUESTED** [review]

**Review by Alice932 — APPROVED** [review]

**Review by olehvavryniv — APPROVED** [review]

**Review by mariadenadel — APPROVED** [review]
Good!

**Review by YuliiaManivska — APPROVED** [review]
Great work!

**Pull request merged**